spec/integration/interactive_mode_spec.rb

```diff
@@ -99,24 +99,46 @@ def expect_to_run_fzf_suggestion(with_pattern:)
     let(:codeowners) { ['lib/new_file.rb @mpospelov @foobar'] }
     let(:owners) { ['@mpospelov'] }
     let(:file_tree) { { 'lib/new_file.rb' => 'bar' } }
-    let(:answers) { ['r', '@mpospelov', 'y'] }
+    let(:answers) { ['r', '@mpospelov', 'a'] }
 
     it 'asks to add new owner to owners' do
       question = <<~QUESTION
         Unknown owner: @foobar for pattern: lib/new_file.rb. Add owner to the OWNERS file?
-        (y) yes
+        (a) add a new owner
         (r) rename owner
         (i) ignore owner in this session
         (q) quit and save
       QUESTION
       expect(runner)
         .to ask(question)
-        .limited_to(%w[y r i q])
+        .limited_to(%w[a r i q])
         .and ask('New owner: ')
         .and ask('Commit changes?')
       expect(codeowners_file_body).to eq("lib/new_file.rb @mpospelov\n")
       expect(owners_file_body.scan('@mpospelov')).to contain_exactly('@mpospelov')
     end
+
+    context 'when the owner`s name was misspelled' do
+      let(:owners) { %w[@mpospelov @foobaz] }
+
+      it 'asks to add new owner to owners' do
+        question = <<~QUESTION
+          Unknown owner: @foobar for pattern: lib/new_file.rb. Did you mean @foobaz? Add owner to the OWNERS file?
+          (c) correct to @foobaz
+          (a) add a new owner
+          (r) rename owner
+          (i) ignore owner in this session
+          (q) quit and save
+        QUESTION
+        expect(runner)
+          .to ask(question)
+          .limited_to(%w[c a r i q])
+          .and ask('New owner: ')
+          .and ask('Commit changes?')
+        expect(codeowners_file_body).to eq("lib/new_file.rb @mpospelov\n")
```

Comment:
I believe we should test the flow when we select `(c) correct to @foobaz` and in that case we can test the changes for the rules
```
expect(codeowners_file_body).to eq("lib/new_file.rb @mpospelov @foobaz\n")
```
WDYT?

Reply:
I agree. Please address it.

Reply:
You're absolutely right! That's the very thing that should be tested. Fixed it